CONTRIBUTING.md

```diff
@@ -11,7 +11,7 @@ Please file a [bug report]. If you are not using the latest version of the libra
 please upgrade and see if that fixes the issue. If not, please create a minimal example
 that demonstrates the bug, and instructions on how to create that setup from a new virtual
 environment. Also include any error tracebacks (unabridged when possible). This will help
-a lot when dignosing the bug. Do not use pictures to include the traceback.
+a lot when diagnosing the bug. Do not use pictures to include the traceback.
```

Comment:
Thanks for spell checking 🎉 Any comments on these instructions?

Reply:
I like them. The key use cases are at the top and details below for those who get that far.

My only advice would be to include an explicit "make workflow" section; it's hinted at in a few places ("make XXXX can do this; make YYYY can do that") but you could achieve more thorough compliance by stating, 

After your code changes are complete, successfully run:
```
make pre-commit
make tests
make tox
```

or whatever you would suggest as the baseline before the PR is submitted.